### PR TITLE
fix for crashing web windows, console search filtering

### PR DESCRIPTION
**Pull request by birarda**

### CODE REVIEW

**Comment by ZappoMan** [comment]
:+1:

**Comment by birarda** [comment]
Need to fix the path to server console for installed shortcuts.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 76d4403737a975c9aa7f83f2fd47db3cd568c5cf available at: https://gist.github.com/9f50fddefc9cb6e26f26<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6908.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6908.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 806d73bfadedcd6a89d14b8919646cc83661457a available at: https://gist.github.com/9f50fddefc9cb6e26f26<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6908.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6908.dmg

File: cmake/macros/SetPackagingParameters.cmake [hunk 1]
```diff
@@ -52,8 +52,17 @@ macro(SET_PACKAGING_PARAMETERS)
...
     set(INTERFACE_INSTALL_DIR ${DMG_SUBFOLDER_NAME})
+
+    set(CONSOLE_APP_CONTENTS "${CONSOLE_INSTALL_APP_PATH}/Contents")
+    set(COMPONENT_INSTALL_DIR "${CONSOLE_APP_CONTENTS}/MacOS/Components.app/Contents/MacOS")
   else ()
-    set(CONSOLE_INSTALL_DIR ".")
+    if (WIN32)
+      set(CONSOLE_INSTALL_DIR "server-console")
+    else ()
+      set(CONSOLE_INSTALL_DIR ".")
```
Comment (jherico) [inline cmake/macros/SetPackagingParameters.cmake]: Is a win32 only solution going to work given that this bug was merged in to the existing one:  https://app.asana.com/0/26225263936266/82945649385427 ?

Comment (birarda) [inline cmake/macros/SetPackagingParameters.cmake]: They aren't duplicates - marked

**Pull request closed without merging**